doc-validation~/com.unity.probuilder@5.0.4.log

```diff
@@ -0,0 +1,311 @@
+Generated for com.unity.probuilder@file:/Users/amylouise/Documents/gitmore/com.unity.probuilder
+Generated on: 2021-09-15 12:22:21 PM
+
+====================================
```

Comment:
Need to update `.npmignore` to include `doc-validation~`

Reply:
Updated (under Debug and source materials)